operator/cmd/mesh/operator-dump.go

```diff
@@ -38,6 +40,8 @@ func addOperatorDumpFlags(cmd *cobra.Command, args *operatorDumpArgs) {
 	cmd.PersistentFlags().StringVarP(&args.common.manifestsPath, "charts", "", "", ChartsDeprecatedStr)
 	cmd.PersistentFlags().StringVarP(&args.common.manifestsPath, "manifests", "d", "", ManifestsFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.common.revision, "revision", "r", "", OperatorRevFlagHelpStr)
+	cmd.PersistentFlags().StringVarP(&args.common.outputFormat, "output", "o", yamlOutput,
+		"Output format: one of json|yaml|flags")
```

Comment:
What's the use case for flags output format of operator dump?
For profile dump, we can see what we have set for each flag, but what's the purpose of flags output for operator dump?

Reply:
@morvencao Fixed. Thanks for the reminder! Also, just created a new issue https://github.com/istio/istio/issues/36568 to track the feature.